web/beacon-app/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,138 @@
+import email from '/src/assets/icons/email.png';
+import github from '/src/assets/icons/github.png';
+import linkedin from '/src/assets/icons/linkedin.png';
+import twitter from '/src/assets/icons/twitter.png';
+import otter from '/src/assets/images/footer-otter.png';
+
+function Footer() {
+  return (
+    <footer className="bg-footer bg-cover bg-no-repeat">
+      <div className="pt-64 2xl:pt-80">
+        <div className="mx-auto max-w-7xl">
+          <div className="grid grid-cols-4 pb-20">
+            <img src={otter} alt="Sea otter" />
+            <div className="font-bold leading-loose">
+              <h3 className="font-['Quattrocento']">PRODUCT</h3>
+              <ul>
+                <li>
+                  <a href="https://rotational.io/ensign">Ensign</a>
+                </li>
+                {/* <li>
```

Comment:
This and the SDK links will be hidden until they are available. A follow up story will be created in Shortcut.